### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by Martinii89** [kaby76/lsp-types#2]
Would love some examples or even a short tutorial on how to use this.

**Comment by kaby76** [comment]
Good idea. There should be several small LSP server examples. I will work on this after I complete the updates from the [spec](https://microsoft.github.io/language-server-protocol/specifications/specification-current/) since October 2020 to the library. There are many.
